### DISCUSSION TITLE
opt to a file diagnostics in summary reporter

### DISCUSSION THREAD

**Discussion by YPetremann** [biomejs/biome#4180] [Ideas]
Actually when running biome check --reporter=summary, it display the Analyser section by rules, but I think an option to display them by files can be interesting

Why ?
Text editors like VSCode reports problems of opened files, so running the tool in summary mode help see which files need to be fixed, and then to open them.

**Comment by ematipico** [comment]
The `--verbose` option should show that information

**Reply by YPetremann** [reply]
I've run "bunx biome check --verbose --reporter=summary"
got the regular summary but not a file list

**Reply by ematipico** [reply]
You need to use the option without the reporter `biome check --write --verbose`

**Reply by YPetremann** [reply]
Could it also run that way with summary reporter ?

**Reply by ematipico** [reply]
Feel free to provide a proposal. At the moment, what you propose isn't actionable because it doesn't explain/show how the files should be shown to the reporter.

**Comment by YPetremann** [comment]
This is the original output of summary
```
Analyzer ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
Some analyzer rules were triggered

Rule Name                                            Diagnostics
lint/nursery/useSortedClasses                        1 (0 error(s), 1 warning(s), 0 info(s))

Checked 52 files in 90ms. No fixes applied.
Found 1 warning.
```
This is the output I propose
```
Analyzer ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
Some analyzer rules were triggered

Rule Name                                            Diagnostics
lint/nursery/useSortedClasses                        1 (0 error(s), 1 warning(s), 0 info(s))

Files                                                Diagnostics
  
/home/.../src/views/ConfigPage.tsx                   1 (0 error(s), 1 warning(s), 0 info(s))

Checked 52 files in 90ms. No fixes applied.
Found 1 warning.
```

As I originaly said, getting a list of files that still have errors and their diagnostics (number of errors, warnings and info)

**Reply by Conaclos** [reply]
I think that you could create your own reporter based on existing reporters and a shell script. For example, you could use the `github` reporter and use a bunch of shell invocation to build your own summary.

**Reply by YPetremann** [reply]
Yeah, already done that, but would not work when called by vite-plugin-checker

**Comment by ianzone** [comment]
I need this feature too, I want to list files that need to be manually fixed, and rather than emit all errors in the console, I would open the IDE to see more detailed error messages.
